internal/resources/fetcher.go

```diff
@@ -38,11 +43,13 @@ type AuditableResources struct {
 }
 
 // NewFetcher returns a new fetcher with a dynamic client
-func NewFetcher(kubewardenNamespace string) (*Fetcher, error) {
+func NewFetcher(kubewardenNamespace string, policyServerFQDN string) (*Fetcher, error) {
 	config := ctrl.GetConfigOrDie()
 	dynamicClient := dynamic.NewForConfigOrDie(config)
-
-	return &Fetcher{dynamicClient, kubewardenNamespace}, nil
+	if policyServerFQDN != "" {
```

Comment:
I think we should assume `policyServerFQDN` is not empty. We should have cobra set a default value when the user does not provide it

Reply:
Thing is, for knowing the FQDN we need to know the service of the PolicyServer pod, hence why it happens later.

We want the in-cluster behaviour as default, so that's why I left it empty, If we want our own (`localhost`, `cluster.local`, whatever the developer uses), we give it value with the flag.

Reply:
I see, thanks